### PR TITLE
feat: don't set job's ttl

### PR DESCRIPTION
**Pull request by zreigz**
keep a succeeded/failed jobs

### CODE REVIEW

File: pkg/client/pipelines.go [hunk 1]
```diff
@@ -132,9 +132,7 @@ func JobSpecFromJobSpecFragment(gateName string, jsFragment *console.JobSpecFrag
 		gateNameAnnotationKey := v1alpha1.GroupVersion.Group + "/gatename"
 		jobSpec.Template.ObjectMeta.Annotations[gateNameAnnotationKey] = gateName
 	}
-	ttlSeconds := int32(7200)
```
Comment (michaeljguarino) [inline pkg/client/pipelines.go:135]: We actually do want a ttl though, it just needs to be long enough that logs are still usable

**Review by michaeljguarino — APPROVED** [review]

File: pkg/client/pipelines.go (hunk 1 above)
Comment (zreigz) [reply pkg/client/pipelines.go:135]: maybe 24h, WDYT?

Comment (michaeljguarino) [reply pkg/client/pipelines.go:135]: Yea that seems sane to me

File: pkg/client/pipelines.go [hunk 2]
```diff
@@ -132,7 +133,8 @@ func JobSpecFromJobSpecFragment(gateName string, jsFragment *console.JobSpecFrag
 		gateNameAnnotationKey := v1alpha1.GroupVersion.Group + "/gatename"
 		jobSpec.Template.ObjectMeta.Annotations[gateNameAnnotationKey] = gateName
 	}
-	
+	jobSpec.TTLSecondsAfterFinished = lo.ToPtr(int32(86400))
```
Comment (michaeljguarino) [inline pkg/client/pipelines.go:136]: could we make the int a const with a descriptive name so its less of a magic number

File: pkg/client/pipelines.go (hunk 1 above)
Comment (zreigz) [reply pkg/client/pipelines.go:135]: done

File: pkg/client/pipelines.go (hunk 2 above)
Comment (zreigz) [reply pkg/client/pipelines.go]: done

**Review by michaeljguarino — APPROVED** [review]

**Pull request merged**